Add reference to YDB SQLAlchemy + Jupyter Notebook example (#12132)
ydb-platform/ydb: ydb/docs/en/core/integrations/gui/jupyter.md

> # Work with {{ ydb-short-name }} from Jupyter Notebook
> 
> Jupyter Notebook - an open-source instrument that provides an ability to create a shareable document that combines computer code, plain language descriptions, data, rich visualizations like 3D models, charts, graphs and figures, and interactive controls.

Reviewer: ```suggestion
# Work with {{ ydb-short-name }} from Jupyter Notebook

[Jupyter Notebook](https://jupyter.org) is an open-source tool for creating shareable documents that combine code, plain language descriptions, data, rich visualizations, and interactive controls.
```

> Dialect [YDB-SQLAlchemy](https://github.com/ydb-platform/ydb-sqlalchemy/releases) allows to work with {{ ydb-short-name }} from instruments like:

Reviewer: ```suggestion
The [ydb-sqlalchemy dialect](https://github.com/ydb-platform/ydb-sqlalchemy/releases) enables working with {{ ydb-short-name }} from tools such as:
```

* [Pandas](https://pandas.pydata.org/)
* [JupySQL](https://jupysql.ploomber.io/)

> Detailed example is available via [notebook](https://github.com/ydb-platform/ydb-sqlalchemy/blob/main/examples/jupyter_notebook/YDB%20SQLAlchemy%20%2B%20Jupyter%20Notebook%20Example.ipynb).

Reviewer: ```suggestion
## Example

A detailed usage example is available as a [notebook](https://github.com/ydb-platform/ydb-sqlalchemy/blob/main/examples/jupyter_notebook/YDB%20SQLAlchemy%20%2B%20Jupyter%20Notebook%20Example.ipynb).```

Prerequisites:

1. Python 3.8+

> 2. [Jupyter Notebook](https://jupyter.org/install#jupyter-notebook)

Reviewer: ```suggestion
2. [Jupyter Notebook](https://jupyter.org/install#jupyter-notebook) or a compatible hosted service
```

> 3. [Existing {{ ydb-short-name }}](https://ydb.tech/docs/ru/quickstart)

Reviewer: ```suggestion
3. Existing {{ ydb-short-name }} cluster, a single-node one from [quickstart](../../quickstart.md) will suffice
4. Network connection between Jupyter Notebook and {{ ydb-short-name }}
```

> To launch example just open notebook and follow each cell.

Reviewer: ```suggestion
To run the example, download the notebook file {% file src="https://raw.githubusercontent.com/ydb-platform/ydb-sqlalchemy/refs/heads/main/examples/jupyter_notebook/YDB%20SQLAlchemy%20%2B%20Jupyter%20Notebook%20Example.ipynb" name="YDB SQLAlchemy - Jupyter Notebook Example.ipynb" %}, open it in Jupyter, and follow each cell sequentially, executing code as necessary.
```
